build_test.go

```diff
@@ -12,11 +12,14 @@ import (
 
 	"github.com/paketo-buildpacks/mri"
 	"github.com/paketo-buildpacks/mri/fakes"
-	"github.com/paketo-buildpacks/packit"
-	"github.com/paketo-buildpacks/packit/chronos"
-	"github.com/paketo-buildpacks/packit/pexec"
-	"github.com/paketo-buildpacks/packit/postal"
-	"github.com/paketo-buildpacks/packit/scribe"
+	"github.com/paketo-buildpacks/packit/v2"
+	"github.com/paketo-buildpacks/packit/v2/chronos"
+
+	//nolint Ignore SA1019, informed usage of deprecated package
+	"github.com/paketo-buildpacks/packit/v2/paketosbom"
```

Comment:
@ryanmoran is it OK to suppress the linting error here from the deprecated `paketosbom` package since we are knowingly doing this?